models/demos/wormhole/stable_diffusion/demo/demo.py

```diff
@@ -103,7 +103,7 @@ def preprocess_images(image_paths):
 
 def run_demo_inference(device, reset_seeds, input_path, num_prompts, num_inference_steps, image_size=(256, 256)):
     disable_persistent_kernel_cache()
-    device.enable_program_cache()
+    # device.enable_program_cache()
```

Comment:
Program cache needs to be enabled for the demo, it will not run well without.

Reply:
There is a hang with program cache enabled, @tt-nshanker, any thoughts as to why?

Reply:
Hang was a one-off. Not a concern now.